lib/open_project/translations/models/i18n_provider.rb

```diff
@@ -85,17 +86,44 @@ def download_locales(output)
     end
   end
 
-  def each_locale
+  def open_locale_zip(force: false)
```

Comment:
Unused method argument - `force`. You can also write as `open_locale_zip(*)` if you want the method to accept any arguments but don't care about them.